roslibrust_gencpp/README.md

```diff
@@ -0,0 +1,19 @@
+# roslibrust_gencpp
+A CLI application and library for generating ROS C++ headers using `roslibrust_codegen` as the backend.
+
+Running the command line application:
+```bash
+roslibrust$ mkdir -p /tmp/sensor_msgs && \ 
+cargo run --bin gencpp -- \
+--msg assets/ros1_common_interfaces/common_msgs/sensor_msgs/msg/BatteryState.msg \
+--package sensor_msgs \
+-I std_msgs:assets/ros1_common_interfaces/std_msgs \
+-I geometry_msgs:assets/ros1_common_interfaces/common_msgs/geometry_msgs \
+-I sensor_msgs:assets/ros1_common_interfaces/common_msgs/sensor_msgs \
+--output /tmp/sensor_msgs
+```
```

Comment:
Per my comment about the `git diff` line in CI, it might be really nice to have this CI be able to detect changes using the diffy library similar to how the other CI works.

Reply:
Moved to cargo test.